src/cookbooks/clients_last_seen_bits.md

```diff
@@ -0,0 +1,136 @@
+# Working with Bit Patterns in Clients Last Seen
+
+An powerful feature of the `clients_last_seen` methodology is that we can capture
+per-day usage information for the entire 28-day period that each row in the table
```

Comment:
This reads a little bit confusing.  How can it be per day and for an entire 28 day period in one row? We are doing a rolling 28 day aggregation here that is tied to a day?  I thought it was fairly flexible to roll up info that occurred over the last 28 days in whatever way you want (could be less than 28 day roll up you just don't have access to things occurring outside of that window?  I always like to call out exactly what makes each row unique when I describe a table.

Reply:
Also calling this out as a limitation of clients last seen (only 28 days) is useful.